source/agora/consensus/data/ConsensusParams.d

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+
+    The set for consensus-critical constants
+
+    This defines the class for the consensus-critical constants. Only one
+    object is being created and being shared in the network. It can be very
+    useful to set the constanst from the start of the network. The
+    consconsensus-critcal constants are the protocol-level contants, so it
+    is only modified for test environments.
```

Comment:
```suggestion
    object should exists for a single node. The `class` is `immutable`, hence
    the constants needs to be set at the start of the process. The
    consensus-critical constants are the protocol-level constants, so they
    shouldn't be modified outside of for test environments.
```

Please re-read documentation. Any simple spell-checker would catch typos such as "constanst", "consconsensus", "critcal" and "contents". A couple typos here and there happens, but that's a lot.

Reply:
> Please re-read documentation. Any simple spell-checker would catch typos such as "constanst", "consconsensus", "critcal" and "contents". A couple typos here and there happens, but that's a lot.

Sorry for the many typos. You had already suggested a plug-in for VS Code but it doesn't work. I will find other plug-in or the way I don't make typos. Thanks.